frontend/src/components/landingPage/MoreInformation.vue

```diff
@@ -0,0 +1,136 @@
+      <div id="moreInfoTextDiv">
+        <p v-if="this.receivedId != this.errorStr">ID : {{ receivedId }}</p>
+        <p v-if="this.groupID != this.errorStr">Group ID : {{ groupID }}</p>
+        <p v-if="this.artifactID != this.errorStr">Artifact ID : {{ artifactID }}</p>
+        <p v-if="this.discovered != this.errorStr">Discovered : {{ discovered }}</p>
+        <p v-if="this.source != this.errorStr">Source : {{ source }}</p>
+        <p v-if="this.version != this.errorStr">Version : {{ version }}</p>
+      </div>
+    </div>
+    <div id="moreInfoTableDiv">
+      <v-data-table :headers="headers" :items="metrics" class="elevation-1"></v-data-table>
+    </div>
+    <app-footer></app-footer>
+  </div>
+</template>
+<script>
+import Header from "../Layout/Header";
+import Footer from "../Layout/Footer";
+import { eventBus } from "../../main";
+
+export default {
+  name: "moreInformation",
+  data() {
+    return {
+      receivedId: String,
+      artifactID: String,
+      groupID: String,
+      discovered: String,
+      source: String,
+      version: String,
+      errorStr: "function String() { [native code] }",
+      metricKeys: [],
+      metricValues: [],
+      metrics: [],
+      headers: [
+        { text: "Metric Name", align: "center", sortable: "true", value: "id" },
+        {
+          text: "Metric Value",
+          align: "center",
+          sortable: "true",
+          value: "values"
+        }
+      ]
+    };
+  },
+  watch : function() {
+    setTimeout(() => {
+      this.startRetrieve();
+    }, 100);
+  },
+  created: function() {
+    eventBus.$on("moreInfoEvent", data => {
+      this.receivedId = data;
+    }),
+    setTimeout(() => {
+      this.startRetrieve();
+    }, 100);
+  },
+  components: {
+    "app-header": Header,
+    "app-footer": Footer
+  },
+  methods: {
+    startRetrieve() {
+      var vm = this;
+      this.$http
+        .get("retrieve/" + vm.receivedId)
+        .then(response => {
+          return response.json();
+        })
+        .then(data => {
+          (vm.groupID = data.messages[0].metadata.groupId),
+            (vm.artifactID = data.messages[0].metadata.artifactId),
+            (vm.discovered = data.messages[0].metadata.discovered),
+            (vm.source = data.messages[0].metadata.source),
+            (vm.version = data.messages[0].metadata.version),
+            (vm.metricKeys = Object.keys(data.messages[0].metricResults));
+          for (var key = 0; key < vm.metricKeys.length; key++) {
+            var metricKey = vm.metricKeys[key];
+            vm.metricValues.push(data.messages[0].metricResults[metricKey]);
+          }
+
+          for (var i = 0; i < vm.metricKeys.length; i++) {
+            vm.metrics.push({
+              id: vm.metricKeys[i],
+              values: vm.metricValues[i]
+            });
+          }
+        }),
+        error => {
+          alert("Invalid results!", error.messages);
+        };
+    },
+    
+  }
+};
+</script>
+<style>
```

Comment:
The layout of this page is currently losing a lot of space unnecessarily... It feels much too loose and much too narrow. Just my thought...

Reply:
#65 Issue has been raised for the same. The layout will change based on the new layout for the results. For example, tree structure.